wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/GenerationEntryPoint.kt

```diff
@@ -69,8 +69,9 @@ private fun generateListOfWrappersForDocs(listOfWrappersInDocs: Path) {
                 writer.println("* $owner")
                 ownedActions
                     .groupBy { it.name }
-                    .forEach { (action, versions) ->
-                        writer.println("    * $action (${versions.joinToString(", ") { it.toMarkdownLinkToKotlinCode() }})")
+                    .forEach { (_, versions) ->
+                        val kotlinClasses = versions.joinToString(", ") { it.toMarkdownLinkToKotlinCode() }
+                        writer.println("    *  ${versions.first().toMarkdownLinkGithub()} - $kotlinClasses")
```

Comment:
Shameless nitpick (but seriously, Markdown can interpret whitespaces in all weird ways, so it's better to be safe):
```suggestion
                        writer.println("    * ${versions.first().toMarkdownLinkGithub()} - $kotlinClasses")
```